bpf/common/http_types.h

```diff
@@ -27,8 +27,8 @@
 // 100K and above we try to track the response actual time with kretprobes
 #define KPROBES_LARGE_RESPONSE_LEN 100000
 
-#define K_TCP_MAX_LEN 256
-#define K_TCP_RES_LEN 128
+#define K_TCP_MAX_LEN 2048
```

Comment:
If we do this, I think we'll have to make it configurable and not the default. I had done early experiments with larger buffers and the performance overhead is very high, for the kinds of workloads we deal with.